src/App.tsx

```diff
@@ -177,8 +177,9 @@ function getMessage(logging: JSON): undefined | getMessageReturn {
             source.push(
               `${logs[msg]["message"]["source"]}`
             )
+            const level = logs[msg]["message"]["level"];
             const text = logs[msg]["message"]["message"];
-            const [debug_level, log_message,level] = logLevel(text);
+            const [debug_level, log_message] = logLevel(text,level);
```

Comment:
Should `debug_level` be renamed to `log_level`?

Reply:
Only issue with changing this to log_level is that we already have the log_level which we will be clashing with the current debug_level. One is the level number (which is needed for the minimum log filtering) and the other is the log level which is shown on the front end. ("WARN", "DEBUG" etc.) Do you have a preference with the two variables being named? maybe lg_level and log_level? Or something like log_level_message?

Reply:
At worst, `log_level_str` is fine